lib/msal-browser/src/error/BrowserConfigurationAuthError.ts

```diff
@@ -31,6 +31,10 @@ export const BrowserConfigurationAuthErrorMessage = {
         desc: "The object passed for the callback was invalid. " +
           "More information is available here: https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/MSAL-basics."
     },
+    inMemRedirectUnavailable: {
+        code: "in_mem_redirect_unavailable",
```

Comment:
If storeAuthStateInCookie = true and In-Memory is selected, ITP cases still work with v2 right? It is obvious I think but want to check all combos. Also a good idea to make a table of config params and supported functionality as a part of this PR.

Reply:
Yes, it should, as the cookies created by MSAL are on the app's domain and thus first-party.

Reply:
will add docs for this in a follow up PR.